### PR TITLE
Feature/modify spreadsheet #624

### PR DESCRIPTION
**Pull request by cullerton**
Script to modify an existing (IRB Document) spreadsheet.
Requires upload_workflow_id, irb_doc_code, cell, and text parameters

### CODE REVIEW

File: crc/scripts/modify_spreadsheet.py [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+        It inserts text into a spreadsheet in the cell indicated.
+        Requires 'irb_doc_code', 'cell', and 'text' parameters.
+        Example: modify_spreadsheet('Finance_BCA', 'C4', 'This is my inserted text')
+        Example: modify_spreadsheet(irb_doc_code='Finance_BCA', cell='C4', text='This is my inserted text')
+        """
+
+    def do_task_validate_only(self, task, study_id, workflow_id, *args, **kwargs):
+        parameters = self.get_parameters(args, kwargs)
+        if len(parameters) == 3:
+            return parameters
```
Comment (danfunk) [inline crc/scripts/modify_spreadsheet.py:38]: What is the point of returning parameters here?

File: crc/scripts/modify_spreadsheet.py [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+
+    def do_task(self, task, study_id, workflow_id, *args, **kwargs):
+        parameters = self.get_parameters(args, kwargs)
+        if len(parameters) == 3:
+
+            spreadsheet = session.query(FileModel). \
+                filter(FileModel.workflow_id == workflow_id). \
+                filter(FileModel.irb_doc_code == parameters['irb_doc_code']).\
+                first()
+            spreadsheet_data = session.query(FileDataModel).\
```
Comment (danfunk) [inline crc/scripts/modify_spreadsheet.py:51]: What if the spreadsheet isn't found?  This seems like a common error point, we should catch it.  What error message would get returned?

File: tests/scripts/test_modify_spreadsheet.py [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+
+        sheet = self.get_sheet(workflow.id, irb_doc_code)
+        self.assertEqual(None, sheet[cell_indicator].value)
+
+        task = workflow_api.next_task
+        self.complete_form(workflow, task, {'cell_indicator': cell_indicator,
+                                            'input_text': input_text})
+
+        sheet = self.get_sheet(workflow.id, irb_doc_code)
+        self.assertEqual(input_text, sheet[cell_indicator].value)
```
Comment (danfunk) [inline tests/scripts/test_modify_spreadsheet.py:56]: This is a great test, well done.

**Review by danfunk — CHANGES_REQUESTED** [review]

File: crc/scripts/modify_spreadsheet.py (hunk 2 above)
Comment (cullerton) [reply crc/scripts/modify_spreadsheet.py]: Added a check to make sure spreadsheet exists.

**Review by danfunk — APPROVED** [review]

**Pull request merged**